inst/odin/lancelot.R

```diff
@@ -366,6 +366,21 @@ n_S_progress[, , ] <- if (j == 1 || n_real_strains == 1)
   rbinom(n_S_progress_tot[i, k], rel_foi_strain[i, j, k]) else
     (if (j == 2) n_S_progress_tot[i, k] - n_S_progress[i, 1, k] else 0)
 
+## Seeding of first wave, we seed in group 4, strain 1, vaccine stratum 1
+##
+seed_step_end <- seed_step_start + length(seed_value)
+seed <- if (step >= seed_step_start && step < seed_step_end)
```

Comment:
consider 
```
seed_mean <- if...
seed <- rpois(seed_mean)
```

then simplifying call to `n_S_progress`

Reply:
have gone with `seed_rate` (we already have `strain_seed_rate`)